waspc/data/Generator/templates/shared/validators.js

```diff
@@ -0,0 +1,23 @@
+export function isValidAbsoluteURL(rawUrl) {
```

Comment:
This is plain JS file, so I can't type it. And since it will live outside the project structure as a plain Node.js script, I wouldn't like to add Typescript here.

Reply:
My bad, thought it was TS